client/src/cmdhftexkom.c

```diff
@@ -32,10 +32,11 @@
 #define TEXKOM_NOISE_THRESHOLD (10)
 
 static inline uint32_t GetGraphBuffer(uint32_t indx) {
-    if (g_GraphBuffer[indx] < -128)
+    int32_t value = get_graph_value_at(indx, true);
```

Comment:
this is just nasty...

Reply:
Can I ask how? I'm genuinely curious about that, mostly because I'm not sure what's nasty about it.

Originally, it was just directly checking the graph buffer at the supplied index. This makes a copy of whatever value is at that index and checks it instead. Wouldn't that be safer to do?